spring-cloud-sleuth-core/pom.xml

```diff
@@ -82,6 +82,11 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-commons</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-aws-messaging</artifactId>
```

Comment:
Not sure if I should bring in the starter of just the code?

Reply:
Concrete dep is ok.